index.js

```diff
@@ -4,6 +4,10 @@
  */
 
 class NetlifyServerPushPlugin {
+  constructor({ redirects }) {
+    this.redirects = Array.isArray(redirects) ? redirects : [];
```

Comment:
Here rather then falling back to `[]` every time I think we should throw an error if the user does not provide `redirects` as Array. i.e we should check 
- if a user provided `redirects` value
	- if its type is Array then assign it to `this.redirects`
	- else throw an error asking the user to provide it as Array.
- else assign `this.redirects` to `[]`
  
What do you say?